src/promptflow/tests/sdk_cli_test/e2etests/test_custom_strong_type_connection.py

```diff
@@ -89,7 +89,7 @@ def test_connection_update(self):
             "configs": {
                 "api_base": "test2",
                 "promptflow.connection.custom_type": "MyCustomConnection",
-                "promptflow.connection.module": "sdk_cli_test.e2etests.test_custom_strong_type_connection",
+                "promptflow.connection.module": "tests.sdk_cli_test.e2etests.test_custom_strong_type_connection",
```

Comment:
Why changed this?

Reply:
Because "__init__. py" has been added, it needs to be modified here.
![image](https://github.com/microsoft/promptflow/assets/23182548/a14f9934-1cb6-410b-9285-bf2e26f5a2fa)